website/www/site/content/en/documentation/programming-guide.md

```diff
@@ -160,6 +160,10 @@ Pipeline p = Pipeline.create(options);
 {{< code_sample "sdks/typescript/test/docs/programming_guide.ts" pipelines_constructing_creating >}}
 {{< /highlight >}}
 
+For a more in-depth tutorial of applying various transforms
```

Comment:
noting that the intro text is the same but the notebook link is different. is that intentional?

Reply:
It was intentional, but probably better to distinguish the terminology. See new commit